src/script_lua.c

```diff
@@ -660,7 +660,7 @@ static void luaReplyToRedisReply(client *c, client* script_client, lua_State *lu
 
         /* Handle double reply. */
         lua_pushstring(lua,"double");
-        lua_gettable(lua,-2);
+        lua_rawget(lua,-2);
```

Comment:
is there a reasonable chance someone would have relied on this?
or are these the same cases that we already recently realized (few that were reported)

Reply:
Updated the top comment and mention that it might be considered a breaking change.